src/components/User.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {users} from '../users';
+
+class User extends React.Component {
```

Comment:
Use a function instead of class.
you didn't use the potentiality of class
https://ru.reactjs.org/docs/components-and-props.html#function-and-class-components